moochub-schema.json

```diff
@@ -1,555 +1,3105 @@
-                          ],
-                          "description": "Either 'Proprietary' or a license identifier according to https://spdx.org/licenses"
-                        },
-                        "url": {
-                          "type": [
-                            "string",
-                            "null"
-                          ],
-                          "format": "iri",
-                          "description": "Links to a license document that applies for the course, preferably as recognized by the SPDX license list"
+                        "inLanguage": {
+                          "type": "string",
+                          "description": "The language the name is given in. Has to be a shortcode according to BCP 47.",
+                          "example": "en"
                         },
                         "name": {
-                          "type": [
-                            "string",
-                            "null"
-                          ],
-                          "description": "A human readable license name"
-                        },
-                        "author": {
-                          "type": [
-                            "string",
-                            "null"
-                          ],
-                          "description": "The author(s) that should be attributed for the work, e.g. when choosing a corresponding CC license such as CC BY-NC-SA"
+                          "type": "string",
+                          "description": "The name of the educational level in the given language by the \"inLanguage\" attribute.",
+                          "example": "INTERMEDIATE"
                         }
-                      }
+                      },
+                      "required": [
+                        "inLanguage",
+                        "name"
+                      ]
                     }
+                  },
+                  "alternateName": {
+                    "type": [
+                      "array",
+                      "null"
+                    ],
+                    "description": "A list of alternative names for the educational level.",
+                    "example": [
+                      "independent use of concept",
+                      "use of concept for oneself"
+                    ],
+                    "items": {
+                      "type": "string"
+                    }
+                  },
+                  "shortCode": {
+                    "type": "string",
+                    "example": "4",
+                    "description": "A short code for the educational level if it is provided by the used framework."
+                  },
+                  "educationalFramework": {
+                    "type": "string",
+                    "description": "The name of the educational framework that describes the educational l",
+                    "example": "DigComp 2.2"
```

Comment:
I would have expected that this is a known set of allowed values, i.e. enum.

Reply:
Yes, it will be as soon as we have made a decision on which frameworks to use. The research is still going on. Should we add a preliminary enum?